gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/routers/requests/requests.py

```diff
@@ -129,19 +141,18 @@ class RagQuery(BaseModel):
                         },
                         'model': 'text-embedding-ada-002',
                     },
-                    'document_index_name': 'my-index-name',
-                    'document_search_params': {
```

Comment:
Filters are still available and used in the current implementation to filter the correct indexing session, we should keep it in the exemple.

Reply:
yes it's a mistake